data/mods/ssb/statuses.js

```diff
@@ -292,6 +292,19 @@ let BattleStatuses = {
 		onFaint() {
 			this.add(`c|%Birdy~!|My last words will always be that I don't spam.`);
 		},
+		// Feather Tuft Innate
+		onModifyDefPriority: 6,
+		onModifyDef(def, pokemon) {
+			if (!pokemon.transformed && !pokemon.illusion) {
```

Comment:
Would pokemon.transformed be true when it forme changes?

Reply:
p sure its when the pokemon is transformed

Reply:
Can this be checked so we can make sure we don't have a sneaky bug on our hand please, maybe add a this.debug call and look for that when testing.

Reply:
It dosent so were good.